### ISSUE TITLE
Strict types conflict if models primary key is integer

### ISSUE THREAD

**Issue by liepumartins** [dive-be/laravel-wishlist#3]
`
Dive\Wishlist\WishCollection::findModel(): Argument #2 ($id) must be of type string, int given, called in /home/vagrant/test/vendor/dive-be/laravel-wishlist/src/WishCollection.php on line 36
`

Happens when route binding or simply calling:
```
Dive\Wishlist\Facades\Wishlist::find('09c09144-42fd-47b2-98b9-396687eb23ca')
```

Likely because the Eloquent model has `$casts = ['id'=>'int'];` for unrelated reasons.

**Comment by mabdullahsari** [comment]
Closed by #4

**Comment by mabdullahsari** [comment]
Fix released as [1.2.1](https://github.com/dive-be/laravel-wishlist/releases/tag/1.2.1). Thanks for the PR!
